sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/telemetry_probes_stats_v1/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH telemetry_probes_with_expiration AS (
+  SELECT
+    channel,
+    probe,
+    type,
+    release_version,
+    last_version,
+    expiry_version,
+    CASE
+      WHEN expiry_version = "never"
+        THEN last_version + 1
+      ELSE IF(
+          last_version + 1 < CAST(expiry_version AS int64),
```

Comment:
What's the difference between last_version + 1 and release_version? Should those be the same? Can we guarantee that version is always `+1`? Think Windows 8 and then Windows 10?

Reply:
release_version is the version it was relesed at 
last_version is the last version the data was seen (which I get from the API)